src/ert/_c_wrappers/job_queue/ext_job.py

```diff
@@ -231,7 +231,7 @@ def __eq__(self, other):
         if self.name() != other.name():
             return False
 
-        if [x for x in self.get_arglist()] != [x for x in other.get_arglist()]:
+        if list(self.get_arglist()) != list(other.get_arglist()):
```

Comment:
Doesn't `get_arglist()` already return `list`?

Reply:
It does indeed! Fixed.